.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "1.0.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Jest: current file",
+      //"env": { "NODE_ENV": "test" },
```

Comment:
Why is this line commented out?

Reply:
It was in the default example, and I left it to remind us that it's an option which we can revisit in the future.